src/inflators/mirror.ts

```diff
@@ -0,0 +1,51 @@
+import { addComponent } from "bitecs";
+import { PlaneGeometry } from "three";
+import { Reflector } from "three/examples/jsm/objects/Reflector";
+import { addObject3DComponent } from "../utils/jsx-entity";
+import { Mirror } from "../bit-components";
+import { HubsWorld } from "../app";
+import { Layers } from "../camera-layers";
+
+const DEFAULT_MIRROR_GEOMETRY = new PlaneGeometry();
+const DEFAULT_TEXTURE_WIDTH = window.innerWidth * window.devicePixelRatio;
+const DEFAULT_TEXTURE_HEIGHT = window.innerHeight * window.devicePixelRatio;
+
+export type MirrorParams = {
+  color?: string;
+};
+
+const DEFAULTS = {
+  color: '#7f7f7f'
+};
+
+export function inflateMirror(world: HubsWorld, eid: number, params: MirrorParams) {
+  params = Object.assign({}, DEFAULTS, params);
+  const geometry = DEFAULT_MIRROR_GEOMETRY;
+  const reflector = new Reflector(geometry, {
+    color: params.color,
+    textureWidth: DEFAULT_TEXTURE_WIDTH,
```

Comment:
This is not new and we don't necessarily need to fix this right now, but I quite dislike that we make this so high resolution... Not sure what a more sane value would be.

Reply:
Yes, probably lower resolution will be better. Added a comment for now.